src/content/docs/setup/install.mdx

```diff
@@ -104,6 +104,24 @@ Running it via Docker can be nice for just trying out kubecolor:
 docker run --rm -it -v $HOME/.kube:/home/nonroot/.kube:ro ghcr.io/kubecolor/kubecolor get pods
 ```
 
+== asdf
+
+For users using the [asdf](https://github.com/asdf-vm/asdf) package manager.
+
+```sh
+asdf plugin-add kubecolor https://github.com/dex4er/asdf-kubecolor.git
+asdf install kubecolor latest
+asdf global kubecolor latest
```

Comment:
Kubecolor seems to be available in the shortname index: https://github.com/asdf-vm/asdf-plugins

So this could probably be reduced to:


```suggestion
asdf install kubecolor latest
```

Reply:
You're absolutely right; asdf can indeed be installed with a simpler command. I've made the corrections. Thank you!